ckanext/canada/templates/snippets/cdts/header.html

```diff
@@ -109,10 +109,6 @@
                 "subhref": {{ recombinant_menu_link('travela') | tojson }},
                 "subtext": {{ recombinant_menu_text('travela') | tojson }},
               },
-              {
-                "subhref": {{ recombinant_menu_link('qpnotes') | tojson }},
-                "subtext": {{ recombinant_menu_text('qpnotes') | tojson }},
-              },
```

Comment:
unrelated change?

Reply:
yes, this is just removing a double entry of qpnotes in the menu